qubes/tests/integ/dispvm.py

```diff
@@ -235,8 +235,12 @@ async def run_preload_proc(self):
             *self.preload_cmd,
             stdout=asyncio.subprocess.PIPE,
         )
-        stdout, _ = await asyncio.wait_for(proc.communicate(), timeout=60)
-        return stdout.decode()
+        try:
+            stdout, _ = await asyncio.wait_for(proc.communicate(), timeout=60)
+            return stdout.decode()
+        except asyncio.TimeoutError:
+            proc.terminate()
```

Comment:
you may want to add `await proc.wait()` after this, to cleanup remains of the process (not doing so may result in zombie processes, which openqa will also report as `cleanup_loop` failure, even if the current issue is caused by something else)